tests/launch_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Oh, that is because I have the repository on a NTFS drive which is mounted as shared folder into an Ubuntu virtual machine.
There are no permissions at all on such a drive, no matter I change them they still appear as 770:
```bash
➜  tests git:(master) ✗ sudo chmod 777 *
➜  tests git:(master) ✗ ls -la
total 13
drwxrwx--- 1 root vboxsf 4096 sty 29 00:50 .
drwxrwx--- 1 root vboxsf 4096 sty 29  2017 ..
-rwxrwx--- 1 root vboxsf  293 sty 28 23:02 launch_tests.sh
-rwxrwx--- 1 root vboxsf 3869 sty 29 00:50 tests.py
```

I will just move the repo to the virtual machine/ubuntu's filesystem and fix the permissions.